src/EPR.Calculator.API.UnitTests/BaseControllerTest.cs

```diff
@@ -687,7 +688,7 @@ protected static IEnumerable<CalculatorRun> GetCalculatorRuns()
             list.Add(new CalculatorRun
             {
                 Id = 2,
-                CalculatorRunClassificationId = 2,
+                CalculatorRunClassificationId = (int)RunClassification.Running,
```

Comment:
Can you make it consistent, either by adding a space in between or removing the space in between :) ?

Reply:
made it consistent